app/Controllers/indexController.php

```diff
@@ -210,6 +215,30 @@ private function updateContext() {
 		FreshRSS_Context::$sinceHours = intval(Minz_Request::param('hours', 0));
 	}
 
+	/**
+	 * Set CSP policies for the main view
+	 */
+	private function setCspPolicies() {
+		$csp_policies = [
+			'default-src' => "'self'",
+			'frame-src' => '*',
+			'img-src' => '* data:',
+			'media-src' => '*',
+		];
+
+		$urls_to_authorize = array_filter(array_map(function ($a) {
+			if (isset($a['method']) && $a['method'] === 'POST') {
+				return $a['url'];
+			}
+		}, FreshRSS_Context::$user_conf->sharing));
+		if ($urls_to_authorize) {
+			$urls = implode(' ', $urls_to_authorize);
+			$csp_policies['connect-src'] = "'self' " . $urls;
```

Comment:
I cannot remember why we need this logic. Could you please refresh my memory @aledeg ?
`connect-src` is not used for Form POST, is it?
https://developer.mozilla.org/docs/Web/HTTP/Headers/Content-Security-Policy/connect-src

https://github.com/FreshRSS/FreshRSS/blob/82851d2039f619f1b2558e06b04a9e47fceeea54/p/scripts/main.js#L909-L915

Reply:
@Alkarex I cannot remember either :'(

Reply:
It was added for Mastodon in afffbfce0, outdated by d3942b296 (there is no longer "POST" share methods).

Reply:
Maybe we can just clean up a bit this part?

Reply:
Yes indeed, I think we should remove this part, and re-consider how to add it next time it becomes needed.

Reply:
Done. I also removed CSP declaration in `rssAction`, I think it's useless